fix: LEAP-1379: disable non-functional export types for MIG projects

robot-ci-heartex opened the pull request:
Hi @jombooth!

This PR was automaticaly generated via Follow Merge. 
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.

- https://github.com/HumanSignal/label-studio-sdk/pull/355 linked by [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/11847131243)

given labeling config
```
<View>
  <Image name="image" valueList="$images"/>
  <RectangleLabels name="labels" toName="image">
    <Label value="Cat"/>
    <Label value="Dog"/>
  </RectangleLabels>
</View>
```

export options are:

![image](https://github.com/user-attachments/assets/296e4ffd-3297-4a68-b3e5-42dcfb309497)

jombooth committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11847131243

jombooth committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11847156217

jombooth commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/11867731677)
> Successfully merged:  66 files changed, 694 insertions(+), 448 deletions(-)

jombooth committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11867720631

jombooth committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11867727471

jombooth committed:
merge develop

jombooth committed:
Merge branch 'fb-LEAP-1379' of github.com:heartexlabs/label-studio into fb-LEAP-1379

jombooth committed:
add unit test for MIG label config export formats

jombooth committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11899232452
